<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgba(0, 0, 0,0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>40</y>
         <width>91</width>
         <height>400</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>362</width>
         <height>400</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
border-bottom-left-radius: 30px;
border-bottom-right-radius: 30px;
border-top-left-radius: 30px;
border-top-right-radius: 30px</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Century Schoolbook</family>
          <pointsize>24</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
font: 75 24pt &quot;Century Schoolbook&quot;;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;LOG IN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="user">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>110</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Century Schoolbook</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0,0,0,0);
border: none;
border-bottom: 2px solid rgba(255, 255, 255, 200);
color: #ffffff;
padding-bottom:7px;
font: 75 12pt &quot;Century Schoolbook&quot;;</string>
        </property>
        <property name="placeholderText">
         <string>Usuario</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ingresar">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>270</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Century Schoolbook</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ingrsar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#ingresar{
background-color: rgba(255, 255, 255,0);
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
border: 1px solid black;
font: 75 12pt &quot;Century Schoolbook&quot;;
color: white;

}

QPushButton#ingresar:hover{
background-color: rgb(234, 234, 234);
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
border: 1px solid black;
font: 75 13pt &quot;Century Schoolbook&quot;;
color: black;

}

QPushButton#ingresar:pressed{
background-color: rgb(40, 40, 40);
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
border: 1px solid black;
font: 75 12pt &quot;Century Schoolbook&quot;;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Iniciar Sesión</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Century Schoolbook</family>
             <pointsize>10</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
font: 75 10pt &quot;Century Schoolbook&quot;;
</string>
           </property>
           <property name="text">
            <string>¿Olvidó su contraseña?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="restablecer">
           <property name="styleSheet">
            <string notr="true">QPushButton#restablecer{
background-color: rgba(255, 255, 255,0);
border-bottom-left-radius: 5px;
border-bottom-right-radius: 5px;
border-top-left-radius: 5px;
border-top-right-radius: 5px;
border: 1px solid black;
font: 75 10pt &quot;Century Schoolbook&quot;;
color: white;

}

QPushButton#restablecer:hover{
background-color: rgb(234, 234, 234);
border-bottom-left-radius: 5px;
border-bottom-right-radius: 5px;
border-top-left-radius: 5px;
border-top-right-radius: 5px;
border: 1px solid black;
font: 75 10pt &quot;Century Schoolbook&quot;;
color: black;

}

QPushButton#restablecer:pressed{
background-color: rgb(40, 40, 40);
border-bottom-left-radius: 5px;
border-bottom-right-radius: 5px;
border-top-left-radius: 5px;
border-top-right-radius: 5px;
border: 1px solid black;
font: 75 10pt &quot;Century Schoolbook&quot;;
color: white;
}</string>
           </property>
           <property name="text">
            <string>Restablecer</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btn_size">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/source/flechaL.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>190</y>
          <width>201</width>
          <height>43</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="password">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>41</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Schoolbook</family>
             <pointsize>11</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgba(0,0,0,0);
border: none;
border-bottom: 2px solid rgba(255, 255, 255, 200);
color: #ffffff;
padding-bottom:7px;
font: 75 11pt &quot;Century Schoolbook&quot;;</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Contraseña</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_pass">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgba(0,0,0,0);
border: none;
border-bottom: 2px solid rgba(255, 255, 255, 200);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Images/recursos.qrc">
             <normaloff>:/source/show.png</normaloff>:/source/show.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>0</y>
         <width>470</width>
         <height>470</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/source/fondo2.jpg);
border-bottom-left-radius: 30px;
border-bottom-right-radius: 30px;
border-top-left-radius: 30px;
border-top-right-radius: 30px</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>70</y>
         <width>211</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/source/logo.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="mensaje">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>250</y>
         <width>361</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 12);
font: 75 14pt &quot;Century Schoolbook&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;“Si todos avanzamos juntos, el éxito llegará solo”. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;— Henry Ford.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>49</y>
         <width>470</width>
         <height>421</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 150);
border-bottom-left-radius: 30px;
border-bottom-right-radius: 30px;
border-top-left-radius: 30px;
border-top-right-radius: 30px</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_salir">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>20</y>
         <width>15</width>
         <height>15</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/source/cerrar.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>400</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/source/MyIcon_white.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>440</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Schoolbook</family>
         <pointsize>7</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
font: 75 7pt &quot;Century Schoolbook&quot;;
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;TECNOLOGY&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>430</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Schoolbook</family>
         <pointsize>7</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
font: 75 7pt &quot;Century Schoolbook&quot;;
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;INNOBYTE&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="hora">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>10</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);
font: 75 9pt &quot;Century Schoolbook&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;00:00:00 -am&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="fecha">
       <property name="geometry">
        <rect>
         <x>351</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Schoolbook</family>
         <pointsize>7</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
font: 75 7pt &quot;Century Schoolbook&quot;;</string>
       </property>
       <property name="text">
        <string>23 de diciembre de 2023</string>
       </property>
      </widget>
      <zorder>frame</zorder>
      <zorder>label_3</zorder>
      <zorder>label_6</zorder>
      <zorder>label_4</zorder>
      <zorder>mensaje</zorder>
      <zorder>btn_salir</zorder>
      <zorder>hora</zorder>
      <zorder>fecha</zorder>
      <zorder>label_8</zorder>
      <zorder>label_9</zorder>
      <zorder>label_5</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Images/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
